### ISSUE TITLE
Не распознается оператор IIF 

### ISSUE THREAD

**Issue by sal0mander** [gretmn102/QSP-VSCode#24]
Описание тернарного оператора есть здесь:  http://wiki.qsp.su/help:conditional

**Comment by gretmn102** [comment]
@sal0mander О, привет, не ожидал, что сюда вообще хоть кто-то напишет. Собственно, что именно не распознается?
![Снимок](https://user-images.githubusercontent.com/14349457/120739057-90318f80-c4f9-11eb-8d7e-a7b607724423.PNG)
Всё вроде бы нормально. Проверял на версии 0.0.6

**Comment by sal0mander** [comment]
@gretmn10 перепроверил. 
Согласен, в случае слитного написания всё ок. В случае использования пробела выдает ошибку распознавания. Пробел движок qsp воспринимает нормально.
![image](https://user-images.githubusercontent.com/29977060/120819377-2eeed800-c55c-11eb-8a77-411f2cecf8f9.png)
![image](https://user-images.githubusercontent.com/29977060/120819487-49c14c80-c55c-11eb-9d3b-c58e1597a9f3.png)
![image](https://user-images.githubusercontent.com/29977060/120819544-5ba2ef80-c55c-11eb-9c0c-6afbee0b61d2.png)

**Comment by gretmn102** [comment]
@sal0mander Да, я тогда думал, что можно *синтаксически* отделить функцию от процедуры вроде такой: `GS 'иди туда, не знаю куда', 'возьми то, не знаю что'`, т.е. когда ставим пробел — то всё, это однозначно процедура, и никакими силами не переубедишь парсер в обратном. Только потом дошло, что QSP определяет по имени, с чем имеет дело, что довольно странно и не очевидно, как по мне. Переделать можно, но это тяжело, да и пока не вижу смысла над этим заморачиваться.

Ах да, у нас лютая путаница в терминах, потому и сливаются функции, операторы, процедуры воедино.
Смотри, немного занудства: оператор — это скорее `+`, `-`, `*` и т.п., у которого есть особенная inline-запись в духе `arg1 + arg2`. Это — бинарный оператор, но в языках программирования есть тернарные операторы, как например в C-подобных языках связка `?` и `:`, т.е. `condition ? thenExpr : elseExpr`, или как в Python:
```python
x, y = 5, 6
print("x" if x > y else "y")
```
Да, в Python тернарные операторы выглядят очень странно, ну да ладно. Так вот, когда ты сказал про тернарный оператор, я уж было подумал, что Byte завез что-то такое, потому я и напрягся.

Перенес вопрос на серверную часть, потому что это — клиент для VS Code, и теоретически сервер можно соединить с любым редактором, который поддерживает [LSP](https://microsoft.github.io/language-server-protocol/overviews/lsp/overview/). Просто VS Code первым под руку попался, на нём, наверное, всё и остановится, хе-хе.
